packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -29,28 +32,22 @@ export const defaultMenuProps = {
 }
 
 /* @vue/component */
-export default VTextField.extend({
+export default mixins(VTextField, Comparable, Filterable).extend({
   name: 'v-select',
 
   directives: {
     ClickOutside
   },
 
-  mixins: [
-    Comparable,
-    Filterable
-  ],
-
   props: {
     appendIcon: {
       type: String,
       default: '$vuetify.icons.dropdown'
     },
     appendIconCb: Function,
     attach: {
-      type: null,
       default: false
-    },
+    } as PropValidator<string | boolean>,
```

Comment:
`Use either a CSS selector string or **an object reference** to the element.`

Reply:
@jacekkarczmarczyk what do you mean?

Reply:
`attach` prop can be `string` or `boolean` or `Element` or maybe `VNode`